docs/cli-reference/gardenadm/gardenadm_token.md

```diff
@@ -22,8 +22,8 @@ Manage bootstrap and discovery tokens for gardenadm join
 ### SEE ALSO
 
 * [gardenadm](gardenadm.md)	 - gardenadm bootstraps and manages autonomous shoot clusters in the Gardener project.
-* [gardenadm token create](gardenadm_token_create.md)	 - Create a bootstrap token on the server
-* [gardenadm token delete](gardenadm_token_delete.md)	 - Delete a bootstrap token on the server
-* [gardenadm token generate](gardenadm_token_generate.md)	 - Generate a random bootstrap token
-* [gardenadm token list](gardenadm_token_list.md)	 - List all bootstrap tokens on the server
+* [gardenadm token create](gardenadm_token_create.md)	 - Create a bootstrap token on the cluster for joining a node
```

Comment:
Should we cover the new commands in e2e tests?

Reply:
https://github.com/gardener/gardener/pull/11942 makes use of some of the `gardenadm token` command. As for most of the others, I don't think it's needed to execute them in e2e tests (they are lightweight enough to run with unit tests for now).